.github/workflows/pr_push.yml

```diff
@@ -38,13 +38,23 @@ jobs:
     needs: [Build]
     uses: ./.github/workflows/reusable_gpu.yml
     with:
-      name: "LEVEL_ZERO"
+      provider: "LEVEL_ZERO"
+      runner: "L0"
       shared_lib: "['ON']"
+  L0-BMG:
+    needs: [Build]
+    uses: ./.github/workflows/reusable_gpu.yml
+    with:
+      provider: "LEVEL_ZERO"
+      runner: "L0-BMG"
+      shared_lib: "['ON']"
```

Comment:
Why the static lib is not tested?

Reply:
I guess it's just to be quicker. It does run for all gpu tests in nightly